src/amo/pages/Home/index.js

```diff
@@ -235,6 +240,14 @@ export class HomeBase extends React.Component {
 
     const showHeroPromo = clientApp !== CLIENT_APP_ANDROID;
 
+    const { promotedExtensions } = shelves;
+    if (Array.isArray(promotedExtensions)) {
+      // If there are fewer than 6 promoted extensions, just use the first 3.
```

Comment:
Interesting... is this a requirement?  If someone's paying for promotion I would have thought we'd always want to show as many as possible.

Reply:
This is from the PRD. I guess they don't want the shelf to look unbalanced with 4 or 5 extensions.